src/server/ftpserver.rs

```diff
@@ -216,6 +220,21 @@ where
         self
     }
 
+    /// Enable proxy protocol.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// use libunftp::Server;
+    ///
+    /// // Use it in a builder-like pattern:
+    /// let mut server = Server::with_root("/tmp").with_proxy_protocol();
+    /// ```
+    pub fn with_proxy_protocol(mut self, proxy_port: u16) -> Self {
```

Comment:
After reading the API Guidelines I realized with_* is really only used if the method creates the object so here you can just name it `proxy_protocol`